src/core_modules/capture-core/components/Pages/Search/SearchPage.container.js

```diff
@@ -8,12 +8,22 @@ import { withErrorMessageHandler, withLoadingIndicator } from '../../../HOC';
 import type { DispatchersFromRedux, OwnProps, Props, PropsFromRedux } from './SearchPage.types';
 import { addFormData } from '../../D2Form/actions/form.actions';
 import { actionCreator } from '../../../actions/actions.utils';
-
+import { searchPageActionTypes } from './SearchPage.actions';
 
 const buildSearchOption = (id, name, searchGroups, searchScope) => ({
     searchOptionId: id,
     searchOptionName: name,
     searchGroups: [...searchGroups.values()]
+        // We use the sorted array to always have expanded the first search group section.
```

Comment:
@JoakimSM also look at this. Now it is being calculated here. This is from the https://github.com/dhis2/capture-app/pull/839#discussion_r462112144. I would prefer to change this in this PR and not in the previous one but you can let me know if this is a blocker for you and I can apply the changes in the previous PR :)

Reply:
Thanks. I was thinking we could move it up even one more level and sort the search groups when the metadata objects are built. Somewhere around here: https://github.com/dhis2/capture-app/blob/d779c06f20e0e5a6ab23b938d9ffe294c7f405e4/src/core_modules/capture-core/metaDataMemoryStoreBuilders/common/factory/searchGroup/SearchGroupFactory.js#L197

Reply:
Dont see why not :) Will do so then in the https://github.com/dhis2/capture-app/pull/839#discussion_r462112144.

Reply:
I would appreciate some of your help here. :) 
How would you go about it, would you do something like the image bellow? Or am I missing a point?
![image](https://user-images.githubusercontent.com/4181674/88932530-e35a0680-d286-11ea-9ed0-8039a932d5f1.png)

Reply:
LGTM 👍